src/ansible_runner/config/_base.py

```diff
@@ -81,7 +81,7 @@ def __init__(self,
                  container_workdir: str | None = None,
                  container_auth_data=None,
                  ident: str | None = None,
-                 rotate_artifacts: int = 0,
+                 rotate_artifacts: int | None = None,
```

Comment:
I'm not seeing a reason for this change. If you revert it back, you can also just revert your change below on line 113, and it will work as you expect.